Source/Core/DolphinWX/AboutDolphin.cpp

```diff
@@ -29,51 +29,51 @@ AboutDolphin::AboutDolphin(wxWindow *parent, wxWindowID id,
 	wxStaticBitmap* const sbDolphinLogo = new wxStaticBitmap(this, wxID_ANY,
 			wxBitmap(iDolphinLogo));
 
-	const wxString Text = wxString::Format(_("Dolphin %s\n"
-				"Copyright (c) 2003-2014+ Dolphin Team\n"
-				"\n"
-				"Branch: %s\n"
-				"Revision: %s\n"
-				"Compiled: %s @ %s\n"
-				"\n"
-				"Dolphin is a GameCube/Wii emulator, which was\n"
-				"originally written by F|RES and ector.\n"
-				"Today Dolphin is an open source project with many\n"
-				"contributors, too many to list.\n"
-				"If interested, just go check out the project page at\n"
-				"https://github.com/dolphin-emu/dolphin/ .\n"
-				"\n"
-				"Special thanks to Bushing, Costis, CrowTRobo,\n"
-				"Marcan, Segher, Titanik, or9 and Hotquik for their\n"
-				"reverse engineering and docs/demos.\n"
-				"\n"
-				"Big thanks to Gilles Mouchard whose Microlib PPC\n"
-				"emulator gave our development a kickstart.\n"
-				"\n"
-				"Thanks to Frank Wille for his PowerPC disassembler,\n"
-				"which or9 and we modified to include Gekko specifics.\n"
-				"\n"
-				"Thanks to hcs/destop for their GC ADPCM decoder.\n"
-				"\n"
-				"We are not affiliated with Nintendo in any way.\n"
-				"GameCube and Wii are trademarks of Nintendo.\n"
-				"The emulator should not be used to play games\n"
-				"you do not legally own."),
-		scm_desc_str, scm_branch_str, scm_rev_git_str, __DATE__, __TIME__);
+	const wxString DolphinText = _("Dolphin");
+	const wxString RevisionText = scm_desc_str;
+	const wxString CopyrightText = _("(c) 2003-2015+ Dolphin Team. \"GameCube\" and \"Wii\" are trademarks of Nintendo. Dolphin is not affiliated with Nintendo in any way.");
```

Comment:
We should probably just say the first year of publication (2003). It's more common, and we wouldn't have to update it every year. Also, IMO spelling out "Copyright" looks prettier than "(c)".